extensions/amp-base-carousel/1.0/arrow.js

```diff
@@ -34,16 +33,17 @@ export function Arrow({customArrow, by, advance, disabled}) {
     }
     advance(by);
   };
+  const styles = useStyles();
+  let classNames =
+    styles.arrowPlacement +
+    ' ' +
+    (by < 0 ? styles.arrowLeft : styles.arrowRight);
+  if (isDisabled) {
+    classNames += ' ' + styles.arrowDisabled;
+  }
+
   return (
-    <div
-      style={{
-        ...styles.arrowPlacement,
-        // Offset button from the edge.
-        [by < 0 ? 'left' : 'right']: '0px',
-        opacity: isDisabled ? 0 : 1,
-        pointerEvents: isDisabled ? 'none' : 'auto',
-      }}
-    >
+    <div class={classNames}>
```

Comment:
Ditto: if we keep the old `style={...}` with a TODO to remove, we can make this pull request merged independently.